inst/include/sampler.h

```diff
@@ -551,6 +548,21 @@ namespace smc {
         }
     }
     
+	template <class Space>
+	void sampler<Space>::Multinomial(unsigned n, unsigned k, arma::vec w, unsigned int * X) {		
```

Comment:
Okay, in that case I should probably change RSIndices to type int in a future version too (I'm using `uRSCount += arma::conv_to<arma::Col<int> >::from(RSIndices)`)

Thanks, I'll keep that in mind for future minor changes like this.